assignments/lambda-classes.js

```diff
@@ -1 +1,188 @@
 // CODE here for your Lambda Classes
+class Person {
+  constructor(obj) {
+    this.name = obj.name;
+    this.age = obj.age;
+    this.location = obj.location;
+    this.gender = obj.gender;
+  }
+  speak() {
+    return `Hello my name is ${this.name}, I am from ${this.location}`;
+  }
+}
+
+class Instructor extends Person {
+  constructor(obj) {
+    super(obj);
+    this.specialty = obj.specialty;
+    this.favLanguage = obj.favLanguage;
+    this.catchPhrase = obj.catchPhrase;
+  }
+  demo(subject) {
+    this.subject = subject;
+    return `Today we are learning about ${this.subject}`;
+  }
+  grade(student) {
+    let percent = Math.random() < 0.5 ? -1 : 1;
+    let gradeChange = Math.round(Math.random() * 15) * percent;
+    student.grade = student.grade + gradeChange;
+
+    console.log(`${student.name} receives a change of ${student.grade} points, and a new grade of ${student.grade}`);
```

Comment:
You are returning `student.grade` twice here - did you mean to console.log the `gradeChange` instead?